src/views/components/edit-model-form.jsx

```diff
@@ -83,16 +87,34 @@ export default class EditModelForm extends React.Component {
             price: e.target.value
         });
     }
+
     changeIsFaulty(e){
         this.setState({
             isFaulty: e.target.value
         });
     }
+
     changeFaultDescription(e) {
         this.setState({
             faultDescription: e.target.value
         });
     }
+
+    changePhoto(e) {
+        let img = document.getElementById('thumbnail-preview');
+        let reader = new FileReader();
+        reader.onload = ((anImgTag) => {
+            return (e) => {
+                let b64StartIdx = e.target.result.indexOf('base64,') + 'base64,'.length;
+                this.setState({
+                    photo: e.target.result.substring(b64StartIdx)
+                });
+                anImgTag.src = e.target.result;
+            };
+        })(img);
+        reader.readAsDataURL(e.target.files[0]);
```

Comment:
If we're storing photos on the server as files, why not just serve that photo directly? e.g. `<img src='/modelphotos/[modeladdress].jpg` />` instead of manually retrieving the photo as base64 on the server, including it in an API call, and then manually parsing that into an image here?